scripts/start.sh

```diff
@@ -18,26 +18,30 @@
 # to install the required packages and configure the system.
 
 oled r
+oled +a "  LITTLE BACKUP "
+oled +b "       BOX      "
+oled +c " by Dmitri Popov"
+oled +R 1
+oled +R 2
+
```

Comment:
This is not useful info, and I'm not that vain to have my name displayed on every boot. :smile: Maybe we should display some useful system info? If you have any suggestions, let me know.